crates/ruff/src/rules/flake8_pyi/rules/ban_doc_strings_in_stubs.rs

```diff
@@ -0,0 +1,27 @@
+use ruff_macros::{define_violation, derive_message_formats};
+use rustpython_parser::ast::{ExprKind, Located};
+
+use crate::ast::types::Range;
+use crate::checkers::ast::Checker;
+use crate::registry::Diagnostic;
+use crate::violation::Violation;
+
+define_violation!(
+    pub struct BanDocStringsInStubs;
```

Comment:
I may rename these to adhere to our new rule naming convention (or feel free to do so if you're up for it): https://github.com/charliermarsh/ruff/blob/main/CONTRIBUTING.md#rule-naming-convention

Basically this would be: `DocstringInStub` (as in, "Allow docstring in stub").

Reply:
done!